Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -859,6 +859,17 @@ def line_problems(line, _lineno)
         problem "Use `depends_on :java` to set JAVA_HOME"
       end
 
+      if line =~ /uses_from_macos (["'](.+)["'])/
```

Comment:
```suggestion
      if line =~ /uses_from_macos ["']([^"']+)["']/
```

Reply:
Can't one capture the first `["']` and then back-reference to it with `\1`?